pyproject.toml

```diff
@@ -30,7 +30,7 @@ dependencies = [
     "protobuf < 3.21",
     "pydantic < 2",
     "pyyaml",
-    "rohmu >= 2.5.0",
+    "rohmu >= 2.7.0",
```

Comment:
@jadkik 
this includes `2.6.0`, https://github.com/Aiven-Open/rohmu/releases/tag/releases%2F2.6.0